bin/phase-helpers.sh

```diff
@@ -1172,6 +1172,10 @@ if ___eapi_has_in_iuse; then
 	}
 fi
 
+if ___eapi_has_version_functions; then
+	source "${PORTAGE_BIN_PATH}/eapi7-ver-funcs.sh" || exit 1
```

Comment:
This is not sufficient to allow to use these new functions in **global scope** of ebuilds.
`ebuild.sh` sources `phase-helpers.sh` only when `[[ $EBUILD_PHASE != depend ]]`.
Probably this code should be added to `ebuild.sh` instead of `phase-helpers.sh`.

(For better performance of generation of metadata cache, it would be even better to inline these new functions in `ebuild.sh` so that a separate file does not need to be sourced...)